contracts/test/integration/IntegrationTestPriceBoundPSMDai.t.sol

```diff
@@ -0,0 +1,341 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import {ERC20, IERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import {SafeCast} from "@openzeppelin/contracts/utils/math/SafeCast.sol";
+import {MockPCVDepositV2} from "../../mock/MockPCVDepositV2.sol";
+import {IPCVDeposit} from "../../pcv/IPCVDeposit.sol";
+import {MockERC20} from "../../mock/MockERC20.sol";
+import {OraclePassThrough} from "../../oracle/OraclePassThrough.sol";
+import {ScalingPriceOracle} from "../../oracle/ScalingPriceOracle.sol";
+import {MockScalingPriceOracle} from "../../mock/MockScalingPriceOracle.sol";
+import {ICore} from "../../core/ICore.sol";
+import {Core} from "../../core/Core.sol";
+import {IVolt, Volt} from "../../volt/Volt.sol";
+import {PriceBoundPSM, PegStabilityModule} from "../../peg/PriceBoundPSM.sol";
+import {getCore, getMainnetAddresses, VoltTestAddresses} from "../unit/utils/Fixtures.sol";
+import {ERC20CompoundPCVDeposit} from "../../pcv/compound/ERC20CompoundPCVDeposit.sol";
+import {Vm} from "./../unit/utils/Vm.sol";
+import {DSTest} from "./../unit/utils/DSTest.sol";
+import {MainnetAddresses} from "./fixtures/MainnetAddresses.sol";
+import {Constants} from "../../Constants.sol";
+
+contract IntegrationTestPriceBoundPSMDaiTest is DSTest {
```

Comment:
Leverage the runner and vip7.sol in these integration tests.

Reply:
https://github.com/volt-protocol/volt-protocol-core/pull/82/files#diff-5e7ab57949bcde3f30345681a85d482c857c906cd0ec820ec6c64a6b6a039a48R115